recipes/aim/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "aim" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/aimhubio/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: af4f06e7ecc3b30537370e732f172670588de580eaa073c7eeed248dec63f248
+
+build:
+  number: 0
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
  number: 0
```

---

That happens when the recipe is missing console_scripts entrypoints, see:

* https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points
* https://github.com/aimhubio/aim/blob/7d31a0feeb2d958cbaf5ed34e410737168a28c3d/setup.py#L133

---

fixed! Thanks for the heads up